staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -500,28 +800,28 @@ func shouldUseNewConnection(t *testing.T) func(httptrace.GotConnInfo) {
 	}
 }
 
-func assertResponse(t *testing.T, resultGot result, statusCodeExpected int) {
+func assertResponseStatusCode(resultGot result, statusCodeExpected int) error {
```

Comment:
What's the point for this change? Why not leave passing testing.T as an argument and calling t.Errorf?

Reply:
Same below

Reply:
for someone looking at a test failure, it will be easier to debug since it will show the exact line number where the error happened. Otherwise, I am fine to revert it.